### PR TITLE
update requirements and environment files for patchmatch 0.1.5

### PR DESCRIPTION
**Pull request by lstein**
- Update pypatchmatch to 0.1.5 (see request from @Kyle0654 here https://discord.com/channels/1020123559063990373/1034740515209486387/1052465462757310464 )

- Removed basicsrs workaround for environment files, now that we know problem was caused by Windows long path issue.

### CODE REVIEW

File: environments-and-requirements/environment-mac.yml [hunk 1]
```diff
@@ -25,10 +25,11 @@ dependencies:
   - diffusers=0.6
   - einops=0.3
   - eventlet
-  - grpcio=1.46
   - flask=2.1
   - flask-socketio=5.3
   - flask-cors=3.0
+  - gfpgan==1.3.8
+  - grpcio=1.46
```
Comment (Kyle0654) [inline environments-and-requirements/environment-mac.yml:32]: I didn't see `grpcio` mentioned in the PR

**Review by Kyle0654 — APPROVED** [review]
Biggest comment is that combining these two changes together in one commit may make it more difficult to unwind if something doesn't work. Otherwise LGTM.

File: environments-and-requirements/environment-mac.yml (hunk 1 above)
Comment (lstein) [reply environments-and-requirements/environment-mac.yml:32]: You were quite right! The change to `gfpgan` broke the CI tests. I'll push a commit that undoes those changes.

Comment (lstein) [reply environments-and-requirements/environment-mac.yml:32]: > I didn't see `grpcio` mentioned in the PR

I reordered it to be in alphabetic order. It was already there.

**Comment by lstein** [comment]
> Biggest comment is that combining these two changes together in one commit may make it more difficult to unwind if something doesn't work. Otherwise LGTM.

I removed the changes to gfpgan. The PR just deals with patchmatch as it should have in the first place.

**Pull request merged**